releasenotes/notes/jmx-fetch-logs-a5ae8bdc27d13a92.yaml

```diff
@@ -0,0 +1,39 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+upgrade:
+  - |
+    List upgrade notes here, or remove this section.
+    Upgrade notes should be rare: only list known/potential breaking changes,
+    or major behaviorial changes that require user action before the upgrade.
+    Notes here must include steps that users can follow to 1. know if they're
+    affected and 2. handle the change gracefully on their end.
+features:
+  - |
+    List new features here, or remove this section.
+enhancements:
+  - |
+    Logs from JMXFetch will now be included in the agent logfile, regardless
+    of the ``log_level`` setting of the agent.
```

Comment:
```suggestion
    Logs from JMXFetch will now be included in the Agent logfile, regardless
    of the `log_level` setting of the Agent.
```

Reply:
@kayayarai this is a rST file, so double-backticks is correct.

We get this correction on _every_ PR that has a changelog -- how could we help your team save the time of making that suggestion?

Reply:
@djmitche Thank you for the info, and apologies for the repeated error. I'll mention it to the team, and add it to our reviewing guidelines! Hopefully that will reduce our over-eagerness.